app/services/food_service.rb

```diff
@@ -0,0 +1,21 @@
+class FoodService
+  class << self
+    def call_for_items(food)
+      response = conn.get("/fdc/v1/foods/search?query=#{food}")
```

Comment:
Similar to your `conn` method, I would set `query` in a block or on a separate line to add readability/maintainability.